mitmproxy/options.py

```diff
@@ -155,6 +155,13 @@ def __init__(self, **kwargs) -> None:
             """
         )
         self.add_option(
+            "stream_websockets", bool, False,
+            """
+            Stream WebSocket frames as and when they are received
+            by the proxy. If True message payload cannot be modified.
```

Comment:
Stream WebSocket **messages between client and server. Messages are captured and cannot be modified.**